src/__test__/components/plots-and-tables/ViolinPlotIndex.test.jsx

```diff
@@ -37,10 +37,12 @@ jest.mock('../../../utils/cacheRequest', () => ({
         const requestedExpression = {};
         body.genes.forEach((geneName) => {
           requestedExpression[geneName] = {
-            min: 0,
-            max: 1.6,
-            expression: [0, 0.4, 0.5, 1.6, 0, 1],
-            zScore: [1, 1.4, 1.5, 2.6, 2, 2],
+            rawExpression: {
+              min: 0,
+              max: 1.6,
+              expression: [0, 0.4, 0.5, 1.6, 0, 1],
+              zScore: [1, 1.4, 1.5, 2.6, 2, 2],
```

Comment:
why is zScore part of the `rawExpression`? I thought that it is some other type of expression. Can you remind me if the values for`min` and `max` also apply to `zScore`?

Reply:
You are right, zScore should be on the same level as rawExpression, not inside.

The test didn't break because violin isn't using zScore)

Reply:
If they are not connected, it is best for them to be on the same level. However, the tests should also represent what the actual structure is. Does the actual structure of the redux store have them on the same level or not?